dropwizard-async-http-client/src/main/java/smartthings/dw/asynchttpclient/AsyncHttpClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package smartthings.dw.asynchttpclient;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AsyncHttpClientConfig {
+	private Map<String, Object> properties = new HashMap<>();
```

Comment:
Are there basic settings we should override by default?  Like to any of the defaults seem unreasonable?  I've not dug into this so the answer might be no.